src/components/tutorialEditor/index.tsx

```diff
@@ -104,7 +129,149 @@ const StyledLabel: AnyStyledComponent = styled.label`
   flex-direction: column;
 `;
 
+const StepContentBody: AnyStyledComponent = styled.div`
+  padding: 40px;
+`;
+
+const StepContentList: AnyStyledComponent = styled.div``;
+
+const ContentBlock: AnyStyledComponent = styled.div`
+  display: flex;
+  border: 1px solid #727272;
+  margin-bottom: 20px;
+`;
+
+const ContentBlockInnerWrapper: AnyStyledComponent = styled.div`
+  padding: 20px;
+  width: ${(props: DeleteProps): string =>
+    props.visible === "flex" ? "93%" : "100%"};
```

Comment:
Might use rem or px instead of percentage so this works nicer on thinner viewports

Reply:
Using percentages within Content Blocks, percentages always add up to 100%.

Tested on multiple view widths.